DOC-3391: Add addButton and addCommand example to creating-a-plugin (#4055)
tinymce/tinymce-docs: modules/ROOT/pages/creating-a-plugin.adoc

= Create a plugin for {productname}
:navtitle: Create a plugin
:description_short: Introducing plugin creation, with an example.

> :description: Create custom TinyMCE plugins using PluginManager.add, editor.ui.registry.addButton, and editor.addCommand. Register plugins, add toolbar buttons, and define commands.

Reviewer: This feels a little odd

```suggestion
:description: Create custom TinyMCE plugins using PluginManager.add, and expand their functionality with custom toolbar buttons using editor.ui.registry.addButton, custom commands using editor.addCommand, and more.
```
?
